internal/workflow/preprocessing.go

```diff
@@ -69,6 +70,34 @@ func (w *PreprocessingWorkflow) Execute(
 		&activities.ValidateStructureParams{SIP: identifySIP.SIP},
 	).Get(ctx, &validateStructure)
 
+	// Add PREMIS objects.
+	var addPREMISObjects activities.AddPREMISObjectsResult
+	e = temporalsdk_workflow.ExecuteActivity(
+		withLocalActOpts(ctx),
+		activities.AddPREMISObjectsName,
+		&activities.AddPREMISObjectsParams{
+			Path:        localPath,
+			ContentPath: identifySIP.SIP.ContentPath},
+	).Get(ctx, &addPREMISObjects)
+	if e != nil {
+		return nil, e
+	}
```

Comment:
This will need considerable refactoring after you rebase the latest work from David. I think you where mobbing with him today, so I hope he provides the necessary feedback to you. Otherwise, let me know if you need to know a little more about what and why it changed.

Reply:
Sounds good! There actually weren't many conflicts when I rebased, but I'll double-check and make sure I'm not missing something.